src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -54,64 +86,131 @@ export default {
     },
 };
 
-export const UnsavedChangesModalWhenChoosingDifferentTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+export const EmptyStateWithBoxAiEnabled: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        fileId: fileWithoutMetadata,
+        metadataSidebarProps: {
+            ...defaultMetadataSidebarProps,
+        },
+    },
+};
+
+export const EmptyStateWithBoxAiDisabled: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        fileId: fileWithoutMetadata,
+        metadataSidebarProps: {
+            ...defaultMetadataSidebarProps,
+            isBoxAiSuggestionsEnabled: false,
+        },
+    },
+};
+
+export const MetadataInstanceEditorWithDefinedTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        fileId: '416047501580',
+        metadataSidebarProps: defaultMetadataSidebarProps,
+    },
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
-        const heading = await canvas.findByRole('heading', { name: 'Metadata' });
-        expect(heading).toBeInTheDocument();
 
-        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' });
-        expect(addTemplateButton).toBeInTheDocument();
+        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 5000 });
         await userEvent.click(addTemplateButton);
 
-        const customMetadataOption = canvas.getByRole('option', { name: 'Custom Metadata' });
-        expect(customMetadataOption).toBeInTheDocument();
+        const customMetadataOption = canvas.getByRole('option', { name: 'My Template' });
         await userEvent.click(customMetadataOption);
+    },
+};
 
+export const MetadataInstanceEditorWithCustomTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        fileId: '416047501580',
+        metadataSidebarProps: defaultMetadataSidebarProps,
+    },
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 5000 });
         await userEvent.click(addTemplateButton);
-        const myTemplateOption = canvas.getByRole('option', { name: 'My Template' });
-        expect(myTemplateOption).toBeInTheDocument();
-        await userEvent.click(myTemplateOption);
 
-        const unsavedChangesModal = screen.getByText('Unsaved Changes');
-        expect(unsavedChangesModal).toBeInTheDocument();
+        const customMetadataOption = canvas.getByRole('option', { name: 'Custom Metadata' });
+        await userEvent.click(customMetadataOption);
+    },
+};
+
+export const MetadataInstanceEditorCancelChanges: StoryObj<typeof MetadataSidebarRedesign> = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const editButtons = await canvas.findAllByRole('button', { name: 'Edit' }, { timeout: 5000 });
```

Comment:
have we confirmed we need the other 7 `, { timeout: 5000 }`?

Reply:
I think that we need them. When running those tests locally they often break without it due to slow loading times. That is why we previously had
```
        await canvas.findByRole('heading', { name: 'Metadata' });
```

to give us lead time for loading the sidebar content.